dotbot_firefox.py

```diff
@@ -96,3 +89,35 @@ def _handle_user_js(self, value: typing.Any) -> bool:
             return True
 
         return link_plugin.handle("link", links)
+
+    def _handle_user_chrome(self, value: typing.Any) -> bool:
+        """Link userChrome.css or chrome directory in each Firefox profile directory."""
+        success = True
+
+        if isinstance(value, str):
+            src = pathlib.Path(os.path.expandvars(os.path.expanduser(value)))
+
+            if src.is_file() and src.name == "userChrome.css":
+                link_plugin = dotbot.plugins.link.Link(self._context)
+                links = {
+                    str(profile / "chrome" / "userChrome.css"): value
+                    for profile in _get_profile_directories()
+                }
+                success &= link_plugin.handle("link", links)
+            elif src.is_dir() and src.name == "chrome":
+                link_plugin = dotbot.plugins.link.Link(self._context)
+                for profile in _get_profile_directories():
+                    chrome_dst = profile / "chrome"
+                    links = {
+                        str(chrome_dst / f.relative_to(src)): str(f)
+                        for f in src.glob("**/*")
+                        if f.is_file()
+                    }
```

Comment:
I have several thoughts about this:

* By default, the Link plugin doesn't create parent directories. On my Windows PC, I don't have a `chrome` subdirectory, so I think this would fail. I think it may be worthwhile for this to create the `chrome` subdirectory if it doesn't exist.
* It seems like `userChrome.css` and `chrome` may need to be mutually exclusive -- if the user specifies one, the other must not be present.
* When `chrome` is specified, it seems like it would be better to link the entire directory, rather than its individual files. What are your thoughts regarding linking the entire `chrome` directory?